Code/JavaWrappers/gmwrapper/GraphMolJava.i

```diff
@@ -49,7 +68,11 @@
 %include <boost/smart_ptr/shared_array.hpp>
 
 /* Include the base types before anything that will utilize them */
-%include "stdint.i"
+#ifdef SWIGWIN
```

Comment:
This mod should also be made to the csharp wrappers. Those can be built on non-windows systems as well (using mono)